manifests/mod/security.pp

```diff
@@ -143,6 +146,7 @@
   String $audit_log_relevant_status                            = '^(?:5|4(?!04))',
   String $audit_log_parts                                      = $apache::params::modsec_audit_log_parts,
   String $audit_log_type                                       = $apache::params::modsec_audit_log_type,
+  Enum['Native', 'JSON'] $audit_log_format                     = $apache::params::modsec_audit_log_format,
```

Comment:
Put the default value here instead.

That way, the default value is correctly documented in the generated documentation.